.travis.yml

```diff
@@ -1,88 +0,0 @@
-language: r
```

Comment:
All this code will be lost in time like tears in rain...

...and good riddance ;)